src/main/java/uk/gov/hmcts/reform/ethos/ecm/consumer/helpers/transfertoecm/TransferToEcmCaseDataHelper.java

```diff
@@ -171,12 +171,14 @@ private static List<RespondentSumTypeItem> createRespondentCollection(
 
     private static List<AddressLabelTypeItem> createAddressLabelCollecton(
         List<uk.gov.hmcts.et.common.model.ccd.items.AddressLabelTypeItem> addressLabelCollection) {
-
+        AddressLabelTypeItem addressLabelTypeItem;
```

Comment:
Why have you defined this here? The variable is only used within the subsequent for loop block.

Reply:
Because of this rule: https://pmd.sourceforge.io/pmd-6.41.0/pmd_rules_java_performance.html#avoidinstantiatingobjectsinloops

This class is however suppressing all PMD warnings but any way I "fixed" this violation here

Reply:
By moving the declaration outside of the loop you've still got the PMD issue! All you've done is move the declaration outside the loop not the instantiation. Please could you revert all these changes so that the declaration is inside the loop too. You can then suppress it

Reply:
reverted that